spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResourceObject.java

```diff
@@ -180,4 +185,10 @@ public OutputStream getOutputStream() throws IOException {
 		}
 		return Channels.newOutputStream(throwExceptionForNullBlob(blob).writer());
 	}
+
+	@VisibleForTesting
```

Comment:
This is actually used by `createBlob()`, so it might make sense to remove `@VisibleForTesting`.

Reply:
Ah yes, will fix.